cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -84,80 +84,83 @@ class FastRandom {
  *
  * @tparam T
  */
-template <typename T>
-concept SimpleRng = requires(T a) {
-                        {
-                            a.next()
-                            } -> std::convertible_to<uint32_t>;
-                    };
+template <typename T> concept SimpleRng = requires(T a)
```

Comment:
The changes in this file and in `log.hpp` look to be format changes. Let's make sure to merge master in once more as I am surprised to file changes here.

Reply:
I did not touch these files, afaik.